server/src/test/java/org/elasticsearch/cluster/routing/allocation/DiskThresholdSettingsTests.java

```diff
@@ -153,6 +153,8 @@ public void testMinimumTotalSizeForBelowLowWatermark() {
         newSettings = Settings.builder()
             .put(DiskThresholdSettings.CLUSTER_ROUTING_ALLOCATION_LOW_DISK_WATERMARK_SETTING.getKey(), "0.50")
             .put(DiskThresholdSettings.CLUSTER_ROUTING_ALLOCATION_LOW_DISK_MAX_HEADROOM_SETTING.getKey(), "500gb")
+            .put(DiskThresholdSettings.CLUSTER_ROUTING_ALLOCATION_HIGH_DISK_MAX_HEADROOM_SETTING.getKey(), "150gb")
```

Comment:
This is one case of settings that this PR would fail. If somebody was setting just one headroom, now they'd need to set all of them explicitly.